### PR TITLE
Fix for allowing login for exemption usernames

### PR DESCRIPTION
**Pull request by Alvarion**

### CODE REVIEW

**Comment by Alvarion** [comment]
Sry for some errors i using Git :)

File: init.lua [hunk 1]
```diff
@@ -34,6 +34,9 @@ local disallowed = {
 }
 
 minetest.register_on_prejoinplayer(function(name, ip)
+	if exemptions[name] then return end
+	
+	-- Check for disallowed and unexemptioned names
```
Comment (ShadowNinja) [inline init.lua]: This only checks for disallowed names (and the word would be "non-exempted").

**Comment by ShadowNinja** [comment]
:+1:  Can't merge ATM though.

File: init.lua (hunk 1 above)
Comment (Alvarion) [inline init.lua]: That's okay, is needed before the discovery of disallowed names allow the name set in minetest.conf as value name = ; if is username admin, owner, Admin, Owner.


I try add this before as first check, but cant login too with reason: "Sorry, someone else is already using this.  Please pick another name.

I must add this line also to function to case insensivity .. i do not known Why but after this , user admin can login to game.

Comment (ShadowNinja) [inline init.lua]: I was talking about the comment :-P

**Comment by ShadowNinja** [comment]
75b96c7cf9083333ba7c7e3dbbbc24e3eb3c3e5e

File: init.lua (hunk 1 above)
Comment (Alvarion) [inline init.lua]: oh :) my fault .. sry :D

**Pull request closed without merging**